assignments/objects.js

```diff
@@ -80,13 +80,11 @@ console.log(intern5.gender);
 // ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 
-// console.log(kennan.speak());
-
-// intern2.speak = function(nameYo) {
-//   console.log("Hello, my name is nameYo");
-// };
+intern2.speak = function(name1) {
+  console.log("Hello, my name is name1");
```

Comment:
Interpolating the value for name1 in the console.log WOULD plug in that value being passed into the function!
just remember to use back ticks ` ` instead of parenthesis :)

console.log(`Hello, my name is ${name1}`);